packages/cypress/src/add-commands/index.ts

```diff
@@ -31,12 +32,7 @@ Cypress.Commands.add(
 
 Cypress.Commands.add(
   'loginToMerchantCenterForCustomView',
-  (
-    commandOptions: Omit<
-      CommandLoginOptions,
-      'entryPointUriPath' | 'initialRoute'
-    >
-  ) => {
+  (commandOptions: TLoginToMerchantCenterForCustomViewCommandLoginOptions) => {
```

Comment:
Thanks!

PS: you also need to update the type declarations here: https://github.com/commercetools/merchant-center-application-kit/blob/d437612a894be32eaa6c124c8e0d1b126148e049/packages/cypress/add-commands/index.d.ts#L22-L28

Reply:
Ah, I overlooked that. Thank you!